### DISCUSSION TITLE
feat(mat-tree): Get the parent tree node from child tree node

### DISCUSSION THREAD

**Discussion by stagefright5** [angular/components#25885] [Ideas]
### Feature Description

 - `MatTreeNode.parent`  should return the parent `MatTreeNode` 

### Use Case

- Useful while computing the node path 
- Avoids multiple tree traversals

**Comment by zarend** [comment]
Hello @stagefright5 ,

Getting access to parent node from child is not prioritized at this time, but I'm going to convert this issue to a discussion.

-Zach
